api/r/CellCensus/tests/testthat/test-open.R

```diff
@@ -0,0 +1,12 @@
+test_that("open_soma", {
+  coll <- open_soma("2023-02-13")
```

Comment:
this build tag will eventually go away (weeks to months from now). The only durable tag (currently) is the `latest` tag.

Reply:
IMO, if tests depend upon having _specific_ data, those tests should use a test fixture (dynamically built, ideally) rather than the live census.  And we should use `latest` if it's the latter case.

Reply:
I think fixtures are ideal but they will probably take quite a bit of time to write and we should probably add them after we ship this MVP. I think using this as a sanity check is a good idea at least for now. Using `latest` might be dangerous since a build could cause tests to start failing in main after a build.

Reply:
Agreed that fixtures are fine for future consideration.

I think it's acceptable if tests fail even if the census data is the cause. These are essentially system tests, rather than unit tests. Consider that many of the Python "unit" tests, which are really system tests, are using pytest markers (annotations) to denote that they depend upon live data.   If we ever have a failure that is census build-specific, it's probably an indication that we're missing an important builder validation. And we can then improve the validator as needed.

Reply:
Agree all. I changed some things to use `latest` where it didn't really matter, and where needed (e.g. checking the respective row of the release directory dataframe), I changed the several occurrences of `2022-02-13` to refer to a single hardcoded constant so that it'll be easy to update if/when that version goes away.